server/controllers/stock/reports/index.js

```diff
@@ -663,6 +689,16 @@ function stockMovementsReport(req, res, next) {
       data.csv = rows;
       data.display = display;
 
+      // group by depot
+      let depots = _.groupBy(rows, d => d.depot_text);
+
+      // make sure that they keys are sorted in alphabetical order
+      depots = _.mapValues(depots, lines => {
```

Comment:
I've never done it, but you could probably use `_.chain()` to make the code look fluid!  Something like this:
```js
const depots = _.chain(depots)
  .groupBy(rows, d => d.depot_text)
  .mapValues(depots, lines => _.sortBy(lines, 'depot_text'))
  .value();
```

Just FYI. :)